### PR TITLE
Move walkhub client configuration to the client module

### PR DESCRIPTION
**Pull request by Coornail**

### CODE REVIEW

**Comment by Coornail** [comment]
Might need some Ux update later.

File: walkhub_client.admin.inc [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ */
+function walkhub_client_configuration_form($form) {
+  $form['configuration']['walkhub_hub_origin'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Walkhub origin'),
+    '#default_value' => variable_get('walkhub_hub_origin'),
+  );
+
+  return system_settings_form($form);
+}
```
Comment (tamasd) [inline walkhub_client.admin.inc]: No newline at the end of the file.

Comment (Coornail) [inline walkhub_client.admin.inc]: =(

Comment (Coornail) [inline walkhub_client.admin.inc]: c09fa23

**Pull request merged**